client/src/components/Forums/ThreadDetails/ThreadDetails.tsx

```diff
@@ -80,18 +105,50 @@ const ThreadDetail = ({ forumId, threadId }: ThreadDetailProps) => {
         <h3 className="font-bold text-2xl">Replies</h3>
         {posts.map((post) => (
           <div key={post._id} className="mb-4">
```

Comment:
This is currently setup to enable users to edit replies they've posted. Is this the goal or are we trying to enable editing of the actual posts themselves?

Reply:
Hey @Mnelson98 - This PR should only allow users to edit the replies they have posted to threads (topics). There is a subsequent PR up that will allow them to edit the initial thread post itself.